strawberryfields/engine.py

```diff
@@ -34,7 +34,9 @@
 # for automodapi, do not include the classes that should appear under the top-level strawberryfields namespace
 __all__ = ["BaseEngine", "LocalEngine"]
 
+
 log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
```

Comment:
Does this count as logging configuration? And if so, should we not be doing it?

Reply:
Thanks for catching that, I'll remove this!